ksql-engine/src/test/java/io/confluent/ksql/codegen/SqlToJavaVisitorTest.java

```diff
@@ -104,7 +104,19 @@ public void shouldProcessArrayExpressionCorrectly() {
         .process(analysis.getSelectExpressions().get(0));
 
     assertThat(javaExpression,
-        equalTo("((Double) ((java.util.List)TEST1_COL4).get((int)(0)))"));
+       equalTo("((Double) ((java.util.List)TEST1_COL4).get((int)0))"));
+  }
+
+  @Test
+  public void shouldProcessArrayNegativeIndexExpressionCorrectly() {
+    final String simpleQuery = "SELECT col4[-1] FROM test1 WHERE col0 > 100;";
```

Comment:
nit: no need for the `WHERE col0 > 100` - makes the test unnecessarily more complicated

Reply:
@agavra updated, it was copy/paste from the _**shouldProcessArrayExpressionCorrectly**_ test.